openapi/openapi/src/main/scala/endpoints/documented/algebra/JsonSchemaEntities.scala

```diff
@@ -0,0 +1,18 @@
+package endpoints
+package documented
+package algebra
+
+import scala.language.higherKinds
+
+/**
+  * Partially applies the [[JsonEntities]] algebra interface to fix the
+  * `JsonRequest` and `JsonResponse` types to be `JsonSchema`.
+  */
+trait JsonSchemaEntities
+  extends JsonEntities
+    with JsonSchemas {
+
+  type JsonRequest[A] = JsonSchema[A]
+  type JsonResponse[A] = JsonSchema[A]
+
+}
```

Comment:
So now the user can choose between `JsonEntities` and `JsonSchemaEntities`. Are there any use cases in which he couldn't go with `JsonSchemaEntites`? In other words, do we still need `JsonEntities`? The only reason I can think of is either for simplicity or for users who really don't want to use `shapeless`.

Reply:
That’s a good point. I will check if it’s possible/easy to remove `JsonEntities`.